### DISCUSSION TITLE
How to manage organizations from service user using API ?

### DISCUSSION THREAD

**Discussion by skillmaker-dev** [zitadel/zitadel#8450] [Q&A] [answered]
Hello, I started working with Zitadel and I want to create organizations and users... using my backend app, I tried to do that using Postman but I'm facing this message "No matching permissions found (AUTH-5mWD2)" whenever I try to create a new organization.

I created a service user as an organization manager of Zitadel org (the one that contains Management-API, Admin-API...), i did the same for the ZITADEL project (project owner global).
I generated the client id and secret and then requested an access token using this: 
`curl --request POST --url http://localhost:8080/oauth/v2/token --header 'Content-Type: application/x-www-form-urlencoded' --data grant_type=client_credentials --data scope='openid profile urn:zitadel:iam:org:project:id:zitadel:aud' --user "mytestapp:RzZ5YSNeeFV1F4aqtelliRadRboNpPfnrUbG5Hy12CfXkbQvcl0yiChNR8485fZv"`

I got the bearer token and used it here: 
![image](https://github.com/user-attachments/assets/6b4a4942-2bdf-4fcd-8d87-e8d74425ba93)

But that didn't work, I can successfully get my org info but not creating orgs:
![image](https://github.com/user-attachments/assets/be6395c0-ecc6-46f6-9d2e-a182d59c1b09)

Any help would be useful, thanks in advance !

**Comment by skillmaker-dev** [comment, marked as answer]
I had to add the service user as a IAM owner by clicking in "Default settings" at top right for it to work.

**Reply by hanspoo** [reply]
Hi, can't find where to do it ¿ can you paste some screen shoot ?

**Comment by hanspoo** [comment]
After creating the Service User, click on plus sign after ZA icon at the top right of the screen, select "Add a Manager", then select the Service User from the list, and finally mark "Iam Owner" checkbox.

**Reply by bamasangkasaportier** [reply]
i've already do that but still return error 

![image](https://github.com/user-attachments/assets/0285dd79-8263-4122-b73e-2273c28c8ccd)

`{
    "code": 16,
    "message": "Errors.Token.Invalid (AUTH-7fs1e)",
    "details": [
        {
            "@type": "type.googleapis.com/zitadel.v1.ErrorDetail",
            "id": "AUTH-7fs1e",
            "message": "Errors.Token.Invalid"
        }
    ]
}`

**Reply by bamasangkasaportier** [reply]
`curl --location 'http://localhost:8080/management/v1/orgs' \
--header 'Content-Type: application/json' \
--header 'Accept: application/json' \
--header 'Authorization: Bearer xx-xx' \
--header 'Cookie: zitadel.useragent=xx=' \
--data '{
  "name": "Customer A"
}'`

**Reply by bamasangkasaportier** [reply]
OH SILLY ME, solved by changing the scope into `[{"type":"text","key":"scope","value":"urn:zitadel:iam:org:project:id:zitadel:aud"}]`
